contracts/test/BeefyClient.t.sol

```diff
@@ -242,6 +242,9 @@ contract BeefyClientTest is Test {
         // Signature count is now 1 after a second submitInitial.
         beefyClient.submitInitial(commitment, bitfield, finalValidatorProofs[0]);
 
+        // Signature count is now 2 after a second submitInitial.
+        beefyClient.submitInitial(commitment, bitfield, finalValidatorProofs[0]);
```

Comment:
To make these tests pass we need another submitInitial to increment the signature count. See comment below on the new math.

Reply:
Guess to trigger the error [here](https://github.com/Snowfork/snowbridge/blob/13b406832cedee0066d9823d4704e6fcab70d258/contracts/src/BeefyClient.sol#L480-L482)?

Just curious how to make the invalid proof in https://github.com/Snowfork/snowbridge/blob/13b406832cedee0066d9823d4704e6fcab70d258/contracts/test/BeefyClient.t.sol#L256 to trigger that error.

Reply:
Maybe not relevant to the change here, but I saw multiple `InvalidValidatorProof` errors in different places in function [verifyCommitment](https://github.com/Snowfork/snowbridge/blob/13b406832cedee0066d9823d4704e6fcab70d258/contracts/src/BeefyClient.sol#L470), would suggest we name the error properly with different names.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1016/commits/cd0034bf59fca50ef38fa8ab9f38fdb66140a35b. I just added an extra error called `InvalidValidatorProofLength` for this case and modified the affected cases.